src/client/Inode.h

```diff
@@ -278,7 +280,7 @@ struct Inode {
       snaprealm(0), snaprealm_item(this),
       oset((void *)this, newlayout->pool_id, this->ino),
       reported_size(0), wanted_max_size(0), requested_max_size(0),
-      _ref(0), ll_ref(0)
+      _ref(0), ll_ref(0), dir_pin(-ENODATA)
```

Comment:
use MDS_RANK_NONE here

Reply:
@ukernel  I use -ENODATA here for the compatibility consideration. 

1. New ceph-fuse, but old MDS:

Ceph-fuse won't get any pin value from old MDS, so `Client::_vxattrcb_dir_pin_exists` can tell getfattr ceph.dir.pin is not supported. So getfattr will return "No such attribute" like before.

2. New ceph-fuse and new MDS, but directory doesn't set pin value:

Ceph-fuse can decode pin value which is -1 and `Client::_vxattrcb_dir_pin_exists` can tell getfattr ceph.dir.pin is already supported. So getfattr will return -1.

If use MDS_RANK_NONE, we can't distinguish above two cases. What do you think?

Reply:
@ukernel  pls ignore above comment. Your suggestion is good.